src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Methods.jl

```diff
@@ -175,6 +175,31 @@ end
 # Normalization                                                        #
 ########################################################################
 
+@doc raw"""
+    is_normal(X::AbsAffineScheme; check::Bool=true) -> Bool
+
+# Input:
+- a reduced scheme ``X``,
+- if `check` is `true`, then confirm that ``X`` is reduced; this is expensive.
+
+# Output:
+Returns whether the scheme ``X`` is normal.
+
+# Examples
+```jldoctest
+julia> R, (x, y, z) = rational_field()["x", "y", "z"];
+
+julia> X = spec(R);
+
+julia> is_normal(X)
+```
```

Comment:
A line of output seems to be missing here.

Reply:
Fixed.